regression/contracts/frees-clause-and-predicates/test.desc

```diff
@@ -0,0 +1,17 @@
+CORE
```

Comment:
if we turn it to KNOWNBUG is it still going to be run as part of contracts regression  ?

Reply:
They will be skipped until you turn then back to `CORE` in the next PR, which implements their correct behavior. Not a blocker for now, but in the future marking as `KNOWNBUG` can help you to avoid updating all regression tests again if you implement a different behavior on another PR.

Reply:
In that case I'd rather keep the test in CORE since it demonstrate that trying to prove a post that depends on the partially implemented `__CPROVER_is_freed` predicate is expected to fail.